extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -338,20 +338,69 @@ export async function promptAndUpdateConnectionStringSetting(projectUri: vscode.
 			return;
 		}
 
-		let existingSettings: (vscode.QuickPickItem)[] = [];
+		// Known Azure settings reference for Azure Functions
+		// https://docs.microsoft.com/en-us/azure/azure-functions/functions-app-settings
+		const knownSettings: string[] = [
+			'APPINSIGHTS_INSTRUMENTATIONKEY',
+			'APPLICATIONINSIGHTS_CONNECTION_STRING',
+			'AZURE_FUNCTION_PROXY_DISABLE_LOCAL_CALL',
+			'AZURE_FUNCTION_PROXY_BACKEND_URL_DECODE_SLASHES',
+			'AZURE_FUNCTIONS_ENVIRONMENT',
+			'AzureFunctionsJobHost__*',
```

Comment:
This isn't the actual name of the setting, rather it's saying that any settings starting with `AzureFunctionsJobHost__` are included. I'd just remove this one for now - maybe something we want to add later on but I doubt it's all that common.